.github/workflows/document.yml

```
@@ -0,0 +1,34 @@
+name: Publish
+on:
+  push:
+    branches:
+      - gh-pages
+    tags:
+      - '*'
+  pull_request:
+  schedule:
+    - cron: 0 0 * * *
+jobs:
+  build:
+    name: Build and Publish
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4
+        with:
+          repository: groonga/groonga
+          path: groonga
+          submodules: recursive
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ruby
+      - name: Install dependencies
+        run: |
+          groonga/setup.sh
+          pip install -r groonga/doc/requirements.txt
+          (cd groonga/doc && bundle install)
+      - name: Build documents
```

英語圏の人たちはdocumentsと言っているような気はするし、groonga.org/docs/だけど、他のところでdocにしちゃったんだよなぁ。どっちに合わせるのがいいかしら。

```suggestion
      - name: Build document
```

---

省略しないやつはあまり気にしたことがなかったですが、省略形は `docs` が多い感じがします。

---

s付きに寄せていくかー